README.md

```diff
@@ -1,12 +1,222 @@
+application policy and an error.
+
+If a pod doesn’t match any policy, the native quota evaluation applies, just like with the standard ResourceQuota.
+
+If a pod matches multiple policies, all matching policies are evaluated, and the results are summed up.
+Please see [label-sidecar.go](https://github.com/kubevirt/application-aware-quota/blob/main/example_sidecars/label-sidecar/cmd/label-sidecar.go) for an example sidecar policy.
+for an example sidecar policy.
+
+### Configure AAQ Operator
+To configure AAQ Operator, set the following fields of the aaq object:
+
+* `aaq.spec.namespaceSelector` - determines in which namespaces AAQ's scheduling gate will be added to pods at creation time. This field follows the standard Kubernetes selector format.
+
+    ***Important Note***: if a namespace selector is not being defined, AAQ would target namespaces with the `application-aware-quota/enable-gating` label as default.
+
+    ***Warning***: Pods that set `.spec.nodeName` are not allowed to use in any namespace that AAQ is targeting. Such pods would fail at creation time. Therefore, users shouldn't be targeting system namespaces where `.spec.nodeName` might be set.
+    
+    For more information please see [this discussion](https://github.com/kubernetes/enhancements/issues/3521#issuecomment-2016957473)
+
+
+* `aaq.spec.configuration.allowApplicationAwareClusterResourceQuota` - (default = false) - allows creation and management of ClusterAppsResourceQuota.
+
+  ***Note***: this might lead to longer scheduling time.
+
+
+* `aaq.spec.configuration.SidecarEvaluators` - Slice of standard Kubernetes container objects. Adding containers to the SidecarEvaluators includes a 
+sidecar calculator for custom counting. These containers should mount a directory under `/var/run/aaq-sockets` called `sockets-dir` and use gRPC to send 
+the custom counting evaluation results to the controller. The setup, including gRPC client/server, is simplified with the [libsidecar](https://github.com/kubevirt/application-aware-quota-api/blob/main/libsidecar/sidecar.go) library, 
+which handles the heavy lifting.
+
+  We've implemented a [simplistic side-car container](https://github.com/Barakmor1/application-aware-quota/tree/adddoc/example_sidecars/label-sidecar) for education purposes. Let's use it as an example.
+  In order to use it, AAQ needs to be defined as follows:
+
+  ```yaml
+  spec:
+    configuration:
+      sidecarEvaluators:
+      - args:
+        - --config
+        - double
+        image: registry:5000/label-sidecar:latest
+        name: sidecar-evaluator
+        volumeMounts:
+        - mountPath: /var/run/aaq-sockets
+          name: sockets-dir
+  ```
+
+  For an example of sidecar implementation using the libsidecar library, 
+  refer to [example_sidecars/label-sidecar](https://github.com/Barakmor1/application-aware-quota/tree/adddoc/example_sidecars/label-sidecar/cmd/label-sidecar.go).
```

Comment:
This points to your personal repo. Is it intentional?

Reply:
Mistake, fixed